Training web/Task01-HTML & CSS/assets/css/style.css

```diff
@@ -0,0 +1,68 @@
+    
+            /* base styles */
+            body {
+                color: #333;
+                margin: 0;
+                font-family: 'Roboto', sans-serif;
+            }
+
+            /* span */
+            .span-header {
+                font-weight: bold;
+                font-size: 24px;
+            }
+
+            /* reusables */
+            .constraint {
+                max-width: 960px;
+                width: 400px;
+                padding: 10px 60px;
+                display: inline-flex;
+            }
+            .button {
+                text-decoration: none;
```

Comment:
text decoration default value is none, I think you dont need this line

Reply:
yes it seems it works the same without that property, gonna make the change.